src/shaders/fill_extrusion.vertex.glsl

```diff
@@ -47,7 +48,11 @@ void main() {
 
     // Add gradient along z axis of side surfaces
     if (normal.y != 0.0) {
-        directional *= clamp((t + base) * pow(height / 150.0, 0.5), mix(0.7, 0.98, 1.0 - u_lightintensity), 1.0);
+        // This avoids another branching statement, but multiplies by a constant of 0.84 if no vertical gradient,
```

Comment:
do we have any idea where the magic -0.84 comes from? maybe nicki would know

Reply:
why would Nicki know? i have no idea where it comes from

Reply:
Nicki worked on the extrusion shaders w lbud

Reply:
@nickidlugash do you remember where this magic number came from?